frontend/src/stories/__snapshots__/storyshots.test.js.snap

```diff
@@ -285,11 +285,12 @@ exports[`Storyshots Crossing List Item Caution 1`] = `
       <div
         className="CrossingListItemFlexContainer"
       >
-        <div
+        <a
```

Comment:
I'm not sure diffing HTML makes sense from a testing perspective. Especially when we're going through a lot of mark up changes as we flesh out this app this can impede workflow. What do you think?

Reply:
I haven't found running the tests and updating snapshots to be too much of a snag in my flow, but if it's causing issues for you we could try to think of another solution.

I'm open to any recommendations.